projects/hslayers/components/layer-manager/widgets/type-widget.component.ts

```diff
@@ -1,19 +1,39 @@
-import {Component} from '@angular/core';
+import {Component, inject, signal} from '@angular/core';
 
-import {HsLanguageService} from 'hslayers-ng/services/language';
 import {HsLayerEditorWidgetBaseComponent} from './layer-editor-widget-base.component';
 import {HsLayerSelectorService} from 'hslayers-ng/services/layer-manager';
+import {HsToastService} from 'hslayers-ng/common/toast';
 
 @Component({
   selector: 'hs-type-widget',
   templateUrl: './type-widget.component.html',
 })
 export class HsTypeWidgetComponent extends HsLayerEditorWidgetBaseComponent {
-  constructor(
-    public HsLanguageService: HsLanguageService,
-    hsLayerSelectorService: HsLayerSelectorService,
-  ) {
+  hsToastService = inject(HsToastService);
```

Comment:
why inject() function instead of more common pattern of DI inside constructor?

Reply:
I just got used to it lately because of more versatility it potentially offers (not binded to constructor, doesnt have to be proviede in super() if class is exteded ) and maybe just like the syntax a bit better, idk :D . But in this particular case it would be better to use constructor to not mix the sytaxes at very least.